docs/source/man_sbws.ini.rst

```diff
@@ -72,6 +72,9 @@ destinations.STR
   verify = BOOL
     Whether or not to verify the destination certificate.
     (Default: True)
+  country = STR
+    ISO 3166-1 alpha-2 country code.
+    Use 00 if the destination URL is a domain name and it is in a CDN.
```

Comment:
Maybe use another alphabetic user-assigned code like `ZZ`? It seems odd to use numeric `00` here when there are still some user-assigned codes available.

Reply:
Changed to ZZ